kolibri/core/device/models.py

```diff
@@ -179,6 +180,35 @@ def secondary_storage_locations(self):
     def secondary_storage_locations(self, value):
         update_options_file("Paths", "CONTENT_FALLBACK_DIRS", value)
 
+    def _get_extra(self, name):
+        """
+        Allows use with `get_device_setting` utility
+        :param name: A str name of the `extra_settings` field
+        :return: mixed
+        """
+        return self.extra_settings.get(name, extra_settings_default_values[name])
+
+    @property
+    def allow_download_on_metered_connection(self):
+        # TODO: fix typo
+        return self._get_extra("allow_download_on_mettered_connection")
```

Comment:
This has been fixed on develop, so we can update this before merge.

```suggestion
        return self._get_extra("allow_download_on_metered_connection")
```